src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -51,6 +55,9 @@ namespace {
 std::string cgroup_v2_app_folder;
 std::string cgroup_v2_system_folder;
 
+// Cgroup folder for the current ray node.
+std::string cgroup_v2_folder;
+
```

Comment:
hm why have these as global variables rather than inside a class definition? the natural interface I was expecting is something like:
```
manager = CgroupManager("/path/to/node_cgroup") # manages everything under the path
manager.MoveProcesses(...)
manager.Cleanup()
```

Reply:
this makes it very easy to swap out the fake implementation for testing and to configure the behavior and subpaths of the manager without mucking around in global variables

Reply:
At the beginning when I made the implementation (two months ago), didn't expect a fake impl and unified interface.